.gitignore

```diff
@@ -31,3 +31,4 @@ package-lock.json
 tsconfig.json
 
 lint.sarif
+kakarot.code-workspace
```

Comment:
why are we ignoring this workspace file:)? could be useful for other vscode users

Reply:
we could maybe, but for now I've put in mine paths like:

```
    "cairols.venvCommand": ". ~/.pyenv/versions/kakarot/bin/activate",
```

so it too specific I guess

Reply:
alright